docs/topics/unsigned-integer-types.md

```diff
@@ -22,19 +28,19 @@ Unsigned types support most of the operations of their signed counterparts.
 >
 {type="warning"}
 
-Same as for primitives, each of unsigned type has corresponding type that represents arrays of that type:
+Same as for primitives, each unsigned type has a corresponding type that represents arrays of that type:
 
-* `UByteArray`: an array of unsigned bytes
-* `UShortArray`: an array of unsigned shorts
-* `UIntArray`: an array of unsigned ints
-* `ULongArray`: an array of unsigned longs
+* `UByteArray`: an array of unsigned bytes.
+* `UShortArray`: an array of unsigned shorts.
+* `UIntArray`: an array of unsigned ints.
+* `ULongArray`: an array of unsigned longs.
 
-Same as for signed integer arrays, they provide similar API to `Array` class without boxing overhead.
+Same as for signed integer arrays, they provide a similar API to the `Array` class without boxing overhead.
 
-When you use unsigned arrays, you'll get a warning that indicates that this feature is not stable yet.
-To remove the warning, opt-in the `@ExperimentalUnsignedTypes` annotation.
+When you use unsigned arrays, you will receive a warning that indicates that this feature is not stable yet.
```

Comment:
```suggestion
When you use unsigned arrays, you receive a warning that indicates that this feature is not stable yet.
```
I see you're expanding the contraction but let's go one step further and get rid of the future tense entirely 😄